invenio_app_rdm/theme/webpack.py

```diff
@@ -20,39 +18,44 @@ def theme():
     This is a callable function in order to lazy load `current_app`
     and avoid working outside application context.
     """
-    return WebpackBundle(
+    return WebpackThemeBundle(
         __name__,
         'assets',
-        entry={
-            'invenio-app-rdm-theme': current_app.config['INSTANCE_THEME_FILE'],
-            'invenio-app-rdm-search-theme': current_app.config[
-                'INSTANCE_SEARCH_THEME_FILE'],
-            'invenio-app-rdm-search-js':
-                './js/invenio_app_rdm/search/SearchMain/index.js',
-            'invenio-app-rdm-searchbar-js':
-                './js/invenio_app_rdm/search/SearchBar/index.js',
-            'invenio-app-rdm-deposits-deposit-searchbar-js':
-                './js/invenio_app_rdm/deposits/RDMDepositSearchBar/index.js',
-            'invenio-app-rdm-deposits-deposit-form-js':
-                './js/invenio_app_rdm/deposits/RDMDepositForm/index.js',
-        },
-        dependencies={
-            # add any additional npm dependencies here...
-            'axios': '^0.19.0',
-            'formik': '^2.1.4',
-            'lodash': '^4.17.15',
-            'luxon': '^1.23.0',
-            'path': '^0.12.7',
-            'prop-types': '^15.7.2',
-            'qs': '^6.9.1',
-            'react': '^16.12.0',
-            'react-dom': '^16.12.0',
-            'react-redux': '^7.1.3',
-            'react-searchkit': '^0.18.0',
-            'redux': '^4.0.5',
-            'redux-thunk': '^2.3.0',
-            "semantic-ui-calendar-react": "^0.15.3",
-            "semantic-ui-css": "^2.4.1",
-            "semantic-ui-react": "^0.88.0"
+        default='semantic-ui',
+        themes={
+            'semantic-ui': dict(
+                entry={
+                    'invenio-app-rdm-theme': current_app.config['INSTANCE_THEME_FILE'],
+                    'invenio-app-rdm-search-theme': current_app.config[
+                        'INSTANCE_SEARCH_THEME_FILE'],
+                    'invenio-app-rdm-search-js':
+                        './js/invenio_app_rdm/search/SearchMain/index.js',
+                    'invenio-app-rdm-searchbar-js':
+                        './js/invenio_app_rdm/search/SearchBar/index.js',
+                    'invenio-app-rdm-deposits-deposit-searchbar-js':
+                        './js/invenio_app_rdm/deposits/RDMDepositSearchBar/index.js',
```

Comment:
PEP8 will complain about these being too long. We can add `# noqa` at the end of the line.